skedulord/__init__.py

```diff
@@ -1 +1 @@
-version = "0.2.5"
+version = "0.2.4"
```

Comment:
The changes to the assets were not being pushed so I messed around a little during debugging so this was changed.